release-notes.hbs.md

```diff
@@ -105,6 +105,8 @@ This release includes the following changes, listed by component and area.
 
 - Remove the field `AuthServer.spec.tls.disabled` and use `AuthServer.spec.tls.deactivated` instead.
 
+- The field `ClientRegistration.spec.redirectURIs` is no longer defaulted to `["http:127.0.0.0:8080"]`.
```

Comment:
**[nit]**: http scheme is malformed, missing `//`

Reply:
Thank you @ddubson! → https://github.com/pivotal/docs-tap/pull/2518